chat.tox.qTox.json

```diff
@@ -4,6 +4,7 @@
 	"sdk": "org.kde.Sdk",
 	"runtime-version": "5.10",
 	"command": "qtox",
+    	"rename-icon" : "qtox",
```

Comment:
Whitespace.

Reply:
This is already fixed with this commit:
https://github.com/flathub/flathub/pull/379/commits/e0c52701ddde009a93c5cfc2551619f8e6405ac7